lib/controller.js

```diff
@@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
         //load view
         lnCmsClientService.getView(endpoint, params)
           .then(function(response) {
-            if (response.data.length > 0)
+            if (response.data.length > 0) {
```

Comment:
is that because of the hack which Cari put in for post? Because afaik it should normally return an object not an array.